Mustknow_2023.md

```diff
@@ -0,0 +1,40 @@
+# Open Science for Physicists
+*Instructions and required information for academic year 2023-24*
+
+This note contains the specific information for the academic year 2023 for the students of OS4P (ns-ph500m) and OSTP (ns-tp506m) who follow the shared part of the two coruses.
+
+For detailed description of the lectures, check [the Syllabus](Syllabus_202x.md). The exercises for this course are team-based. See [the list of assignments](Exercises_2023.md) for the full list.
+
+## Welcome message on Blackboard
+We are excited about this first edition of the "Open Science for Physicists". This interactive course consists of 8 lectures, given by us or external speakers, and exercises which will be performed and discussed during the sessions. Because of this interactive format, participation in the lectures are mandatory and online participation cannot be accomodated. If you are hindered on the first week, please contact the course assistant, Roosmarijn Knopper r.l.knopper@uu.nl, as soon as possible.
+
+Lectures 1, 2, 3, and 8 are common with the "Open Science for Theoretical Physics" course (ns-tp506m). The commonalities and differences between these two sister courses will be explained in more details during the first lecture.
+
+The course syllabus and exercises will be posted on the course repository (more information will follow). The crucial announcement will be posted on the Blackboard.
+
+All the lectures are scheduled on Monday mornings from 9:00-12:45, starting on 11 September 2023. 
+The venue is [the Learning Plaza](https://www.uu.nl/en/education/future-learning-spaces/learning-spaces/learning-plaza) in the Minnaert building (former canteen).
+
+Sanli Faez, Erik van Sebille, Tanja Hinderer
+
+## What to expect? 
+The OS4P course is about real-life skills and experiences of researchers in a rapidly evolving academic landscape. With the advancement of open science, not only we need an updated look at "how science works", but also need to know which tools are available and which skills will be necessary. Therefore, this course is a combination of lectures and hands-on exercises to get a good feeling of these new developments. 
+Some of the open science practices are still not widely embraced by some factions of the scientific community. These frictions will naturally influence the post-graduate career of our students. To appreciate and form an opinion on these practices, the students are sometimes deliberately faced with uncertain circumstances, just like the real world. 
```

Comment:
I find 'factions' rather aggressive term. Would the more neutral 'some people in the scientific community' not be better?

Reply:
adjusted! see next PR